src/content/ui-api/kubectl-gs/update-cluster.md

```diff
@@ -19,38 +19,46 @@ user_questions:
 
 # `kubectl gs update cluster`
 
-This command allows to upgrade a workload cluster. Upgrades can either be triggered immediately, or the upgrade can be scheduled to happen at a specific date and time.
+This command's purpose is to modify details of a workload cluster. Currently it allows to upgrade a workload cluster to a newer [release]({{< relref "/general/releases" >}}).
 
 ## Usage
 
-### Upgrade a cluster immediately
+### Upgrading a workload cluster {#cluster-upgrade}
+
+Upgrades can either be triggered immediately, or the upgrade can be scheduled to happen at a specific date and time.
+
+The following example shows how to upgrade a workload cluster to the release specified via the `--release-version` flag.
 
 ```nohighlight
 kubectl gs update cluster \
-  --name a1b2c \
+  --provider aws \
   --namespace org-acme \
-  --release-version 16.1.0 \
-  --provider aws
+  --name a1b2c \
+  --release-version 16.1.0
 ```
 
-### Schedule a cluster upgrade
+Note that all flags are mandatory. As a result of the command execution, the cluster resource's version label will be updated, which in turn triggers the cluster upgrade.
```

Comment:
Imo it would be worth to spell out the `cluster resource's version label` in case anyone wants to upgrade their clusters with vanilla kubectl

Reply:
Done. It's `release.giantswarm.io/version`.